cmd/bridge/main.go

```diff
@@ -445,12 +439,37 @@ func main() {
 		Handler: srv.HTTPHandler(),
 	}
 
-	log.Infof("Binding to %s...", httpsrv.Addr)
+	klog.Infof("Binding to %s...", httpsrv.Addr)
 	if listenURL.Scheme == "https" {
-		log.Info("using TLS")
-		log.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
+		klog.Info("using TLS")
+		klog.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
 	} else {
-		log.Info("not using TLS")
-		log.Fatal(httpsrv.ListenAndServe())
+		klog.Info("not using TLS")
+		klog.Fatal(httpsrv.ListenAndServe())
+	}
+}
+
+// helper function converts string values to appropriate string number values
+// Also, this translate the old capnslog values that are currently sent by Console Operator
+func convertLogLevel(val string) string {
```

Comment:
I'd change the the log level arg to take the new values instead of trying to translate. The command-line flags aren't user-facing API, so we have the ability to change them. It will require operator changes, though.

Reply:
I have a change already in place on the operator side that will send the right values.  I just wanted to prevent any issues in the interim,

Reply:
We can leave this temporarily while we update the operator if needed.

Reply:
> I have a change already in place on the operator side that will send the right values. I just wanted to prevent any issues in the interim,

OK makes sense

Reply:
Console Operator PR250 changes the values coming over: https://github.com/openshift/console-operator/pull/250